doc/source/user_guide/solution.rst

```diff
@@ -28,15 +37,33 @@ The ``solve.set/gradient_scheme`` TUI sets the gradient options.
     
 Selecting solution controls 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The ``solve.set.p_v_controls`` TUI sets pressure-velocity controls.
+The following example shows a comparison between the TUI command and the
+python code for selecting the pressure velocity controls.
+
+TUI command
+
+.. code:: scheme
+
+    /solve/set/p-v-controls 0.3 0.4
+
+Python command
 
 .. code:: python
 
     session.solver.tui.solve.set.p_v_controls(0.3,0.4) # Momentum and Pressure
 
 Creating report definitions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The ``solve.report_definitions`` TUI enters the report definitions menu.
+The following example shows a comparison between the TUI command and the
+python code for report definitions.
```

Comment:
```suggestion
Python code for report definitions.
```

Reply:
Does there need to be a verb? (... Python code for generating report definitions.)